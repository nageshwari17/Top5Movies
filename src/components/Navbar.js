import React from "react";

const Navbar = () => {
  return (
    <nav className="nav-wrap red">
      <div className="container">
        <a href="/#" className="brand-logo">
          Top Movies
        </a>
      </div>
    </nav>
  );
};

export default Navbar;
