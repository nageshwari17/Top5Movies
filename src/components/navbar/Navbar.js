import React from 'react';

const Navbar = () => {
  return (
    <nav className="nav-wrap red" data-testid="navbar">
      <div className="container">
        <a href="/#" className="brand-logo">
          Top Movies
        </a>
      </div>
    </nav>
  );
};

export default Navbar;
