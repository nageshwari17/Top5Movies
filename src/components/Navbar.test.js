import React from 'react';
import ReactDOM from 'react-dom';
import Navbar from './Navbar';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';

afterEach(cleanup);

it( 'renders without crashing', () => {
  const div = document.createElement('nav');
  ReactDOM.render(<Navbar></Navbar>, div);
});

it('renders navbar correctly', () => {
  const { getByTestId } = render(<Navbar ></Navbar>);
  expect(getByTestId('navbar')).toHaveTextContent('Top Movies');
});

it('match snapshot', () => {
  const tree = renderer.create(<Navbar />).toJSON();
  expect(tree).toMatchSnapshot();
});