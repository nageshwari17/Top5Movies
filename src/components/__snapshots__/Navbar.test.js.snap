// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`match snapshot 1`] = `
<nav
  className="nav-wrap red"
  data-testid="navbar"
>
  <div
    className="container"
  >
    <a
      className="brand-logo"
      href="/#"
    >
      Top Movies
    </a>
  </div>
</nav>
`;
